deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -52,25 +55,178 @@ def upload(options, screenshots, max_n_threads = 16)
         localizations = version.get_app_store_version_localizations
       end
 
-      upload_screenshots(screenshots_per_language, localizations, options, max_n_threads)
+      upload_screenshots(updated_sets_per_language, localizations, options, max_n_threads)
```

Comment:
The `updated_sets_per_language` should default to `screenshots_per_language` in case the `overwrite_screenshots` option had been set (see other comment).

Reply:
I decided to leave it like it is due to the disregarding of the `overwrite_screenshots` flag. In case we decide it is not the way we want to do things, I will update this. 😃